sql/plan/ddl.go

```diff
@@ -308,6 +321,20 @@ func (d *DropTable) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter, error)
 		}
 	}
 
+	if len(d.triggerNames) > 0 {
+		//TODO: if dropping any triggers fail, then we'll be left in a state where triggers exist for a table that was dropped
```

Comment:
Also need to sort them here to find the order to drop them. At least a TODO, or implement the sort

Reply:
Actually that's not necessary, I'm dumb :p

Reply:
Great catch!! This would have bitten us at some point in the future lol

Reply:
It's still worthwhile in the event that a deletion fails. That way the existing triggers are still in a parsable state, so it's definitely a good idea...at least until we get statement-level transaction-style support or whatever that would be called lol

Reply:
But also, I guess it's okay if the triggers are broken due to deleting a table and a trigger randomly failing deletion. You can (and should) still manually go and delete all of the triggers that didn't delete automatically. So yeah I guess it's overall a non-issue.